modules/calib3d/src/undistort.dispatch.cpp

```diff
@@ -508,14 +507,6 @@ static void cvUndistortPointsInternal( const CvMat* _src, CvMat* _dst, const CvM
     }
 }
 
-void cvUndistortPoints(const CvMat* _src, CvMat* _dst, const CvMat* _cameraMatrix,
```

Comment:
Currently, this is only used internally, it is not part of the public C API (which does not have much in calib3d, cf https://github.com/opencv/opencv/blob/4.x/modules/calib3d/include/opencv2/calib3d/calib3d_c.h). This PR switches the C code that uses it to C++, so there is no need for that function anymore.

Reply:
for example https://github.com/shimat/opencvsharp uses cvUndistortPoints. shouldn't this be preserved?

Reply:
```
git grep cvUndistortPoints
```
dies not return anything in their repo, where do they use it? They use `cv::undistortPoints` which is the C++ API. `modules/calib3d/src/calib3d_c_api.h` is an internal header so modifying it fine, other projects cannot use the API it offers.

Reply:
https://github.com/imclab/opencvsharp/blob/master/src/OpenCvSharp/Src/PInvoke/NativeMethods.cs#L483 (frankly i am not an expert but i think some projects still use OpenCV's C api )

Reply:
I agree, the C API is till used, but the API removed in this PR is INTERNAL to OpenCV, nobody outside of OpenCV is using it because they cannot use it: the header that declares it is not public, it is in `src`, not in `include`. `cvUndistortPoints` was removed from the public C API 6 years ago in https://github.com/opencv/opencv/commit/8f15a609afc3c08ea0a5561ca26f1cf182414ca2#diff-7ae6742689a7ee66932275c29e27989687a7b9e6ca2549a765f5b27414c753daR53

The link you just sent me is from a different repo, the last commit on that file is "[support 2.4.10](https://github.com/imclab/opencvsharp/commit/f55344d7c5d93838206c9a55153be4636fa99734)" which is a very old OpenCV that is not supported anymore.